dbms/src/Storages/KVStore/MultiRaft/RegionCFDataBase.h

```diff
@@ -44,16 +73,18 @@ struct RegionCFDataBase
     using Status = bool;
 
     static const TiKVKey & getTiKVKey(const Value & val);
-
     static const TiKVValue & getTiKVValue(const Value & val);
 
-    RegionDataRes insert(TiKVKey && key, TiKVValue && value, DupCheck mode = DupCheck::Deny);
+    RegionDataMemDiff insert(TiKVKey && key, TiKVValue && value, DupCheck mode = DupCheck::Deny);
 
     static size_t calcTiKVKeyValueSize(const Value & value);
-
     static size_t calcTiKVKeyValueSize(const TiKVKey & key, const TiKVValue & value);
```

Comment:
Change the line 315 in `Region::tryCompactionFilter` from 
```
    data.cf_data_size -= RegionWriteCFData::calcTiKVKeyValueSize(write_map_it->second);
```
to
```
    data.cf_data_size -= RegionWriteCFData::calcTotalKVSize(write_map_it->second).payload;
```

... and then we can move these two methods to be "private"

Reply:
move `tryCompactionFilter ` into RegionData